plugins/polio/js/src/domains/Calendar/campaignCalendar/Filters.js

```diff
@@ -113,9 +116,34 @@ const Filters = ({
     const theme = useTheme();
     const isLargeLayout = useMediaQuery(theme.breakpoints.up('md'));
     const [textSearchError, setTextSearchError] = useState(false);
-
+    const filtersFilled = useCallback(() => {
+        return (
+            countries ||
+            search ||
+            roundStartFrom ||
+            roundStartTo ||
+            showOnlyDeleted ||
+            campaignType ||
+            campaignGroups ||
+            orgUnitGroups
+        );
+    }, [
+        campaignGroups,
+        campaignType,
+        countries,
+        orgUnitGroups,
+        roundStartFrom,
+        roundStartTo,
+        search,
+        showOnlyDeleted,
+    ]);
     useEffect(() => {
         setFiltersUpdated(true);
+        if (filtersFilled()) {
+            setFilterLaunched(true);
+        } else {
+            setFilterLaunched(false);
+        }
```

Comment:
This looks a bit overkill to use a local state and change it with a useeffect. You can set the url parma to true while you execute the search instead no ?

Reply:
Thank you. Good observation